### ISSUE TITLE
Esercenti che offrono consegna a domicilio a Roma

### ISSUE THREAD

**Issue by avivace** [emergenzeHack/covid19italia#320]
**Priorità**: bassa

## Richiesta
https://www.deliveryroma.it/lista-esercenti/

## Risolto?

* [ ] Ho verificato che il problema è stato risolto

**Comment by stefanoauciello** [comment]
file estratto, da capire se va bene il formato.

[data.txt](https://github.com/emergenzeHack/covid19italia/files/4396343/data.txt)

**Comment by iltempe** [comment]
@avivace per questo c'era un'esigenza specifica? volevamo popolare una pagina? @cristigalas @favoeva @dontyna dove possiamo mettere questi dati?

**Comment by avivace** [comment]
Grazie @stefanoauciello ! Purtroppo non ho idea di come processare il file che hai generato.. Per ogni esercizio commerciale serve Nome, indirizzo, link (almeno). Quindi per ognuno di essi generare una riga con queste informazioni, separate da un carattere (come la `,`). Una lista di link in quel modo purtroppo non è utilizzabile (a meno che non ho capito come vuoi fare)

**Comment by stefanoauciello** [comment]
@avivace ok chiaro, ma il codice che scrivo per generare il file vi serve? oppure vi serve solo il file per importarlo staticamente? perchè nel caso lo faccio nel linguaggio che conosco di più.

**Comment by avivace** [comment]
Fallo come preferisci! In ogni caso se lo condividi poi si può riutilizzare/prendere come spunto per realizzare altri "importer"

**Comment by stefanoauciello** [comment]
ok, certo lo condivido allora appena riesco lo faccio e lo condivido, pensavo fosse da importare nel sito in qualche modo e che quindi dovesse rispettare un linguaggio.

**Comment by avivace** [comment]
No, l'importante é che il risultato finale sia processabile (CSV o JSON)

**Comment by favoeva** [comment]
> @avivace per questo c'era un'esigenza specifica? volevamo popolare una pagina? @cristigalas @favoeva @dontyna dove possiamo mettere questi dati?

in questa issue  #260 si era deciso per il form

**Comment by skyauciellos** [comment]
Ciao, ho usato il "|" per dividere i dati perchè negli indirizzi ecc erano presenti le virgole, in ogni caso vi allego l'output e il codice Java che può essere riutilizzato.

[output.txt](https://github.com/emergenzeHack/covid19italia/files/4398753/output.txt)
[Script.txt](https://github.com/emergenzeHack/covid19italia/files/4398757/Script.txt)

**Comment by skyauciellos** [comment]
il csv è cosi organizzato :
Nome negozio
Link
Telefono
Indirizzo
Zona di consegna

**Comment by avivace** [comment]
@skyauciellos nella prima riga, se metti il nome di ogni campo posso parsarlo automaticamente

**Comment by stefanoauciello** [comment]
Ok, guarda se va bene, intanto aggiorno lo script.

[output.txt](https://github.com/emergenzeHack/covid19italia/files/4399653/output.txt)
[Script.txt](https://github.com/emergenzeHack/covid19italia/files/4399655/Script.txt)

**Comment by cristigalas** [comment]
@iltempe qui secondo me si può chiudere.
